src/generator/operations.ts

```diff
@@ -266,7 +266,7 @@ function generateOperation(clientName: string, op: Operation, imports: ImportMan
   }
   // TODO Exception for Pageable LRO operations NYI
   if (isLROOperation(op)) {
-    text += `func (client *${clientName}) Begin${op.language.go!.name}(${params}) (${returns.join(', ')}) {\n`;
+    text += `func (client *${clientName}) ${op.language.go!.methodPrefix}${op.language.go!.name}(${params}) (${returns.join(', ')}) {\n`;
```

Comment:
I think you can move this line back to its original place and do a check to see if `methodPrefix` has been defined then add it; this way it will work for methods other than LROs (see also change on line 666).

Reply:
I'm thinking of not adding an if. If methodPrefix is nil then nothing will be appended. Thoughts?

Reply:
Done. I left it with the if for now, it provides more control.

Reply:
Even better.